src/Google.Api.Gax/Api/Gax/ClientHelper.cs

```diff
@@ -24,8 +24,8 @@ public class ClientHelper
         private readonly CallSettings _globalCallSettings;
 
         /// <summary>
-        /// Constructs a helper from the given settings. These are expected to be cloned
-        /// already, and not changed afterwards.
+        /// Constructs a helper from the given settings.
+        /// Behaviour is undefined if settings are changed afterwards
```

Comment:
Bring back the full stop :)
(I'd also say "after construction" rather than just "afterwards")